src/navx_node.cpp

```diff
@@ -2,25 +2,50 @@
  * navx_node.cpp
  * Runs the Kauai Labs NavX, using modified NavX library
  * VERSION: 1.0.0
```

Comment:
You forgot to bump version numbers for this PR. Usually, 1.X.X bump is for major (breaking or just big) changes, X.1.X is for normal changes, and X.X.1 is (for small projects like ours) documentation changes, or moving stuff around, with no "real" change to code, but bigger projects will use this for bugfixes as well, since there tend to be much more of them.